db_master_deploy/deploy.sh

```diff
@@ -203,8 +203,9 @@ copy_database() {
     # add some metainformation to the copied database as comment
     psql -U pgkogis -d template1 -h localhost -c "COMMENT ON DATABASE ${target_db} IS 'copied from ${source_db} on $(date '+%F %T') with command ${COMMAND} by user ${USER}';" > /dev/null
 
-    # set database to read-only if it is not a _master database
-    if [[ ! ${target} == master ]]; then
+    # set database to read-only if it is not a _master or _demo database
+    REGEX="^master|demo$"
```

Comment:
Not sure if i understand the question ``^ $``are indicating the start/end of the string.
The following patterns are working:
```bash
target=demo && regex="^master|demo$" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="master|demo" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="^(master|demo)$" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

target=demo && regex="(master|demo)" && if [[ ${target} =~ ${regex} ]]; then echo true; else echo false; fi
true

```